pkg/clicmd/root.go

```diff
@@ -80,18 +81,25 @@ func appRoot() (string, error) {
 
 // parseCommand does an early parse of the command line and returns
 // what will ultimately be recognized as the command by cobra.
-func parseCommand(args []string) (string, error) {
+func parseCommand(args []string) (string, bool, error) {
 	fset := pflag.NewFlagSet("", pflag.ContinueOnError)
 	fset.ParseErrorsWhitelist.UnknownFlags = true
 	fset.BoolP("help", "h", false, "") // Needed to avoid pflag.ErrHelp
 	if err := fset.Parse(args); err != nil {
-		return "", err
+		return "", false, err
 	}
 	if len(fset.Args()) == 0 {
-		return "", nil
+		return "", false, nil
 	}
 
-	return fset.Args()[0], nil
+	flags := []string{"-h", "--help"}
```

Comment:
This is not necessary - pflag is already parsing the command line.
The `BoolP` method call above returns a `*bool` that points to a boolean representing the help flag. You can capture that and just return its underlying value to the caller.

Reply:
Removed superfluous code and used BoolP